media/stylus/home.styl

```diff
@@ -21,7 +21,11 @@ $home-involved-background-y = 50px;
     minimize-header-search();
 
     .home-callouts {
-        background: #4d4e53;
+        background: #fff;
+
+        .column-container {
+            padding: $grid-spacing;
```

Comment:
Clashes with code on line 466 to give twice as much padding as we need on tablet and mobile view. 465 to 467 can probably just get deleted (but I can't leave a comment on those lines because they weren't edited).